### ISSUE TITLE
Add explicit feeRate option for sendmany RPC method

### ISSUE THREAD

**Issue by yurayakimenko** [bitcoin/bitcoin#19662]
**Is your feature request related to a problem? Please describe.**
It's always hard to send a bunch of transactions with sendmany method using conf_target option. Cause build-in fee estimator not suggesting required fee rate for our purposes. How could I specify transaction fee in BTC/kB for sendmany without creating raw transaction (building outputs and inputs manually)?

**Describe the solution you'd like**
Add fee_rate option to sendmany RPC method.

**Describe alternatives you've considered**
Currently I'm trying to estimate required conf_target based on mempool size and making weird algorithms that limiting that value someway.

**Additional context**
That is not so convenient for us. So the better solution would be to provide fee_rate option that is much easier to manipulate. Thanks in advance!

**Comment by MarcoFalke** [comment]
This is being worked on, see #19543

**Comment by yurayakimenko** [comment]
@MarcoFalke my feature request is not about normalizing units, it's about adding `fee_rate` option to `sendmany` RPC method!

**Comment by MarcoFalke** [comment]
#11413 has been merged, #19543 is the second step

**Comment by yurayakimenko** [comment]
@MarcoFalke #11413 would be in release version only after #19543 would be done?

**Comment by jonatack** [comment]
@yurayakimenko #11413 is merged into master and will normally be part of the next release 0.21 scheduled toward the end of this year, according to the schedule in the pinned issue https://github.com/bitcoin/bitcoin/issues/18947. This month I will propose follow-ups to 11413 to be part of that release.

**Comment by jonatack** [comment]
You can use #11413 now in master; I recently used the explicit feerate option on master with sendtoaddress to make a near-instant yet relatively inexpensive payment on a busy mempool day and it worked very well.

**Comment by jonatack** [comment]
This feature request has been delivered, e.g. with #11413 and #20305, and ISTM this issue can be closed.
